### PR TITLE
DOCS-3292 Android Mobile Vitals Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Small doc edit for Android Mobile Vitals.

### Motivation

DOCS-3292

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

**Review by charlesbaiden — COMMENTED** [review]
hey Austin, LGTM - Just to confirm, we're going to have the same for iOS and ReactNative, right? Also I think we should wait for a couple of weeks before we push this to prod so that we can get the picture of the new Mobile Vitals widget, what do you think? Otherwise, thanks for updating the docs! :)

File: docs/rum_mobile_vitals.md [hunk 1]
```diff
@@ -20,8 +22,8 @@ The following metrics provide insight into your mobile application's performance
...
+| CPU ticks per second           | High CPU usage impacts the [battery life][9] on your users’ devices.  <br /><br />  RUM tracks CPU ticks per second for each view and the CPU utilization over the course of a session. The recommended range is <40 for good and <60 for moderate. <br /><br />  You can see the top views with the most number of CPU ticks on average over a selected period time under **Event Timings and Mobile Vitals**.                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (maycmlee) [inline docs/rum_mobile_vitals.md:25]:
```suggestion
| CPU ticks per second           | High CPU usage impacts the [battery life][9] on your users’ devices.  <br /><br />  RUM tracks CPU ticks per second for each view and the CPU utilization over the course of a session. The recommended range is <40 for good and <60 for moderate. <br /><br />  You can see the top views with the most number of CPU ticks on average over a selected time period under **Event Timings and Mobile Vitals**.                                                                                                                                                                                                                                                                                                                                                        |
```

File: docs/rum_mobile_vitals.md [hunk 2]
```diff
@@ -20,8 +22,8 @@ The following metrics provide insight into your mobile application's performance
...
+| Memory utilization             | High memory usage can lead to [OutOfMemoryError][10], which causes the application to crash and creates a poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate. <br /><br />  You can see the top views with the most memory consumption on average over a selected period time under **Event Timings and Mobile Vitals**.                                                                                                                                                                                                                                                                                            |
```
Comment (maycmlee) [inline docs/rum_mobile_vitals.md:26]:
```suggestion
| Memory utilization             | High memory usage can lead to [OutOfMemoryError][10], which causes the application to crash and creates a poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate. <br /><br />  You can see the top views with the most memory consumption on average over a selected time period under **Event Timings and Mobile Vitals**.                                                                                                                                                                                                                                                                                            |
```

**Review by maycmlee — APPROVED** [review]
Just a couple of small edits!

**Review by xgouchet — APPROVED** [review]

**Review by mariusc83 — APPROVED** [review]

**Pull request merged**